custom_components/venta/humidifier.py

```diff
@@ -169,6 +169,14 @@ async def async_set_mode(self, mode: str) -> None:
             action = {"FanSpeed": int(mode[-1])}
         await self._send_action(action)
 
+    async def _send_action(self, data: dict[str, Any]) -> None:
+        """Send action to device."""
+        await self._device.action(self._map_to_action(data))
```

Comment:
I've done small update on main branch regarding the default handling of sending action. Can you have a look if this works fine?